### PR TITLE
Benchmark immutadot and display results on front page

### PR DESCRIPTION
**Pull request by frinyvonnick**
### Prerequisites
- [x] I have read the [Contributing guidelines](https://github.com/Zenika/immutadot/blob/master/.github/CONTRIBUTING.md)
- [x] I have read the [Code of conduct](https://github.com/Zenika/immutadot/blob/master/.github/CODE_OF_CONDUCT.md) and I agree with it

### Description
Benchmark immutadot with immer's benchmark and display results in README

**Issue :** #194

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -190,6 +190,18 @@ toLowerCase({ nested: { prop: { 1: { msg: 'Hello' }, 2: { msg: 'Hi' }, 3: { msg
 
 When applying operations on a path immutad●t tries to create the minimum of objects or arrays needed to guarantee your data structure to be immutable.
 
+[mweststrate](https://github.com/mweststrate) made a [simple benchmark](https://github.com/Zenika/immutadot/tree/master/packages/immutadot-benchmark/src/performance.spec.js) on the performance of [immer](https://github.com/mweststrate/immer) and other methods for setting a property on multiple items of a todo list and keep it immutable. Here are the results with immutad●t in addition:
```
Comment (nlepage) [inline README.md]: I think we should mention here some information on hardware (MacBook Pro...) and software (node 8, without parallelism)...

File: README.md [hunk 2]
```diff
@@ -190,6 +190,18 @@ toLowerCase({ nested: { prop: { 1: { msg: 'Hello' }, 2: { msg: 'Hi' }, 3: { msg
 
 When applying operations on a path immutad●t tries to create the minimum of objects or arrays needed to guarantee your data structure to be immutable.
```
Comment (nlepage) [inline README.md]: Put that under benchmark, this is a memory related information, memory is less relevant than speed to JS developers...

**Comment by nlepage** [comment]
Should we merge like this or add more benchmarks ?

**Comment by frinyvonnick** [comment]
I think we can add more benchmarks later maybe about number of instantiate objects ?

**Comment by nlepage** [comment]
Fine with me :+1:

**Pull request merged**